mypy.ini

```
@@ -15,4 +15,5 @@ no_implicit_optional = True
 warn_return_any = True
 no_implicit_reexport = True
 strict_equality = True
-warn_redundant_casts = True
\ No newline at end of file
+warn_redundant_casts = True
+exclude = venv/
```

https://github.com/Chia-Network/chia-blockchain/blob/35540012d74829149f8dda56663b860eb1a6b828/mypy.ini.template#L2

```suggestion
include = chianft,tests,*.py
```